python/cudf/cudf/core/groupby/groupby.py

```diff
@@ -597,6 +597,33 @@ def func(x):
 
         return self.agg(func)
 
+    def describe(self):
+        """Return descriptive stats of the values in each group"""
+
+        def func1(x):
+            return x.quantile(0.25)
+
+        def func2(x):
+            return x.quantile(0.5)
+
+        def func3(x):
+            return x.quantile(0.75)
+
```

Comment:
We should rename these function names to something like `_quantile_0.25`, `_quantile_0.5` and have these defined outside this method so that they are no re-defined every time this API is called.